ddp-workspace/projects/ddp-dsm-ui/src/app/services/dsm.service.ts

```diff
@@ -875,6 +874,15 @@ export class DSMService {
     );
   }
 
+  public getPhiReport(realm: string, guid: string, orderNumber: string): Observable<any> {
```

Comment:
Similar comment here to phi-model-manifest in terms of specificity of return type.